.github/ISSUE_TEMPLATE/launch-scope.md

```diff
@@ -0,0 +1,15 @@
+Description
+Define the tasks that will lead to a successful launch of this project, create tickets for those tasks, and address any known unknowns pertaining those tasks.
+
+Proposed Solution
+- [ ] Go through the `Launch - *` issue templates, identify the ones that are relevant to this project, and create new issues with them, modifying them as necessary.
+- [ ] Coordinate with the project manager and technical lead on what this means for the project (timeline, budget, scope, etc).
+- [ ] Describe what “launch” consists of for this project in broad terms. Post this summary in this ticket and ensure that the whole team is on board and understands it. We may want to share this summary in other places such as Basecamp as well. Some examples:
+
+> This project will be launched in a brand new production environment within the client’s own Platform.sh account. Once the project has been QAed and approved by the client, the example.com domain’s DNS records will need to be updated by the client’s IT staff.
+
+> Launching this project will consist of enabling new features X, Y, and Z in the client’s existing Acquia production environment after said features have been QAed and approved by the client in a staging environment. After enabling these features, migrations A, B, and C will need to be manually triggered and monitored by Chromatic.
+
+> The launch process will begin by granting access to this repo to the client’s internal devops staff, and sending an email message to the team with instructions on how to build all of the necessary assets. The client’s devops staff is responsible for the eventual deployment, which they will schedule ahead of time. Chromatic will be available for support.
+
+> The launch of this project will consist of the merging of the feature branch PR on 3/31 at a time posted in Basecamp. Following the merge, the client will perform several manual content clean-up tasks.
```

Comment:
I would only suggest that we indent those examples so that they are rendered as children of the checkbox item:
```suggestion

    > This project will be launched in a brand new production environment within the client’s own Platform.sh account. Once the project has been QAed and approved by the client, the example.com domain’s DNS records will need to be updated by the client’s IT staff.

    > Launching this project will consist of enabling new features X, Y, and Z in the client’s existing Acquia production environment after said features have been QAed and approved by the client in a staging environment. After enabling these features, migrations A, B, and C will need to be manually triggered and monitored by Chromatic.

    > The launch process will begin by granting access to this repo to the client’s internal devops staff, and sending an email message to the team with instructions on how to build all of the necessary assets. The client’s devops staff is responsible for the eventual deployment, which they will schedule ahead of time. Chromatic will be available for support.

    > The launch of this project will consist of the merging of the feature branch PR on 3/31 at a time posted in Basecamp. Following the merge, the client will perform several manual content clean-up tasks.
```

I _think_ that should work.